doc/developer-guide/api/functions/TSTrafficServerVersionGet.en.rst

```diff
@@ -55,6 +55,8 @@ Example
 
     #define PLUGIN_NAME "hello_world"
 
+    DbgCtl dbg_ctl{PLUGIN_NAME};
```

Comment:
Do you want to establish the convention of putting these in anonymous namespaces here?

Reply:
I think it's not that simple.  If I add an anonymous namespace, `check_ts_version()` probably belongs in it also.  But, if I make such changes consistently, in all the files I'm touching, it will likely double the size of this PR.  I get a lot of surprising push back in reviews on this project.  For example, push back that seems to dispute the value of RAII.  Which makes me feel like adding any sort of cleanup risks adding further delay to merging PRs.

Reply:
I just mean in this example, since folks tend to copy pasta examples.  If we want to prefer anonymous namespaces, having that in the example will help with proliferation.